controllers/collector_controller.go

```diff
@@ -87,6 +87,8 @@ func (r *CollectorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		switch col.Spec.CollectorType {
 		case "ecr_event":
 			c = collector.NewEcrEventCollector(r.Log, r.AWSConfig, col.Spec.ECR.QueueName)
+		case "harbor_event":
```

Comment:
nitpick: I kinda want to name this to just `harbor`, and rename the ECR collector to just `ecr`.